apps/eth/views.py

```diff
@@ -39,9 +39,17 @@ def _process_event(event):
         except MultipleObjectsReturned as exc:
             LOG.info(f"MultipleObjectsReturned during get/get_or_create for event {event['transaction_hash']}: {exc}")
         except ObjectDoesNotExist:
-            log_metric('transmission.info', tags={'method': 'events.create', 'code': 'non_ethaction_event',
-                                                  'module': __name__})
-            LOG.info(f"Non-EthAction Event processed Tx: {event['transaction_hash']}")
+            if event['event_name'] == 'Transfer':
```

Comment:
Technically there can be multiple event subscriptions to different contracts that emit events with the same name. There could be two contracts that emit 'Transfer', in which case this could potentially log non-ShipToken events. Right now that is _not_ the case, but I think we should try to play it safe.

@mlclay was talking about maybe adding a 'project' field to the incoming Event data from Engine. If this is added, we could use that to filter things down here to only events from the `ShipToken` event subscription.

Reply:
Is there another way to check that atm? Or is there some type of validation I could do before logging?

Reply:
I talked to Lucas about it, we're going to have a task next sprint for adding support to Engine for this, and then fixing this issue.